src/main/kotlin/g0301_0400/s0381_insert_delete_getrandom_o1_duplicates_allowed/RandomizedCollection.kt

```diff
@@ -0,0 +1,57 @@
+package g0301_0400.s0381_insert_delete_getrandom_o1_duplicates_allowed
+
+// #Hard #Array #Hash_Table #Math #Design #Randomized
+// #2022_11_24_Time_1313_ms_(50.00%)_Space_148_MB_(25.00%)
+
+@Suppress("kotlin:S2245")
+class RandomizedCollection() {
```

Comment:
Please reuse java solution

https://github.com/javadev/LeetCode-in-Java/blob/main/src.save/main/java/g0301_0400/s0381_insert_delete_getrandom_o1_duplicates_allowed/RandomizedCollection.java

Reply:
It is only faster than 25%